src/app/Services/authService/auth.service.ts

```diff
@@ -54,6 +54,7 @@ export  class  AuthService {
      this.saveAuth(token, expiresAt.toLocaleString(), res.user, res.userName, res.groupId, res.profilePicId);
      this.userAuthListener.next(res.user);
      } else {
+      localStorage.setItem('message',res.message);
```

Comment:
No need for this .  You can simply use this 
```suggestion
  this.userAuthListener.next(res.message);
```
and remove this line from if and else blocks.and add it after the else block